src/pages/guides/FAQ/FAQ -11-24

```diff
@@ -0,0 +1,57 @@
+# Frame V4 API FAQ
+
+The Frame V4 API allows developers to integrate Frame.io’s surface for creatives and their teams to manage their work for every step of the asset delivery lifecycle from plan to deliver.
+
+* How do I **authenticate**?
+
+    The first step in accessing the [Frame.io](http://frame.io/) API is getting authenticated. Create the user authentication credentials and then the access token. Adobe Identity Management Service (IMS) is Adobe’s OAuth 2.0-based identity management feature that supports authentication. There are two steps required for generating the access token:
+
+    * The user has to authorize the application before the token generation. The details for this can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#authorize-request).
+    * The next step is fetching the access token. The above step returns a **code** in the response body. This is used in this step. The exact endpoints to trigger and the parameters to use can be found [here](https://developer.adobe.com/developer-console/docs/guides/authentication/UserAuthentication/IMS/#fetching-access-tokens).
+* Where is the **API reference**?
+    * Go [here](https://developer-stage.adobe.com/frameio/api/current/) to see the Frame V4 API reference pages. 
```

Comment:
We should investigate whether this can be done as a relative link or injected using environment variables so it will work both in the staging and production environments.

Reply:
Thanks Jeff. I'll look to you to lead that effort and report back!